zebra-state/src/service.rs

```diff
@@ -694,8 +694,8 @@ impl StateService {
             rsp_rx
         };
 
-        // We've finished sending finalized blocks when:
-        // - we've sent the finalized block for the last checkpoint, and
+        // We've finished sending semantically verrified blocks when:
+        // - we've sent the semantically verrified block for the last checkpoint, and
```

Comment:
This comment is about ending checkpoint verification and starting semantic verification, it describes the checkpoint end condition.

```suggestion
        // We've finished sending checkpoint verified blocks when:
        // - we've sent the verified block for the last checkpoint, and
```

Reply:
(This is the comment that the auditors really needed to read!)